XRTK-Core/Packages/com.xrtk.core/Editor/Utilities/UnityFileHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace XRTK.Editor.Utilities
+{
+    public static class UnityFileHelper
+    {
+        private static readonly string[] UnityFileExtensions = {
+            "*.meta",
+            "*.mat",
+            "*.anim",
+            "*.prefab",
+            "*.unity",
+            "*.asset",
+            "*.guiskin",
+            "*.fontsettings",
+            "*.controller",
+        };
+
+        public static List<string> GetUnityFiles(string assetsRootPath)
```

Comment:
Should we also make sure this utility can handle relative/absolute paths?

Reply:
Are we expecting full paths back or relative paths back?

Reply:
Also maybe a better name would be `GetUnityAssetsAtPath` or `GetUnityAssetsInDirectory` and pass in a `DirectoryInfo` instead?

Reply:
Everywhere this is used, we only have a string path.  Might consider an overload to use DirectoryInfo, but nowhere would use it at present.  So this is fine.

OK, Will rename to GetUnityAssetsAtPath